Driverless AI/get-started-with-open-source-custom-recipes-tutorial/get-started-with-open-source-custom-recipes-tutorial.md

```diff
@@ -187,36 +187,36 @@ This dataset set has 3333 customers(rows) and 21 columns representing attributes
 
 *Things to Note:*
 
-- **Under summary**: The validation score is 0.9071 with a best accuracy of 0.9304 (click on ROC then hover over Best ACC)
+- **Under summary**: The validation score is 0.9080 with a best accuracy of 0.9286 (click on ROC then hover over Best ACC)
 
-- **Variable Importance**: The top variable that led to a customer churning according to this model was the total amount a customer was being charged for their "Day Charge." The other variable of most importance is the total amount the customer is being charged for their evening calls,  "Eve calls." Intuitively this makes sense because if a customer is overpaying for calls in the morning when they might be the most active and also overpaying for their evening calls, it could have led a dissatisfied customer.
+- **Variable Importance**: The top variable that led to a customer churning according to this model was "CustServ Calls." The other variable of most importance is "Day_charge". Intuitively this makes sense because if a customer is overpaying for calls in the morning when they might be the most active and also overpaying for "CustServ Calls", it could have led to a dissatisfied customer.
 
 ### Deeper Dive and Resources
 
 - [Kaggle’s Churn in Telcom’s Dataset](https://www.kaggle.com/becksddf/churn-in-telecoms-dataset)
 
 ## Task 3: Recipe: Transformer
 
-In Task 2, we ran an experiment to build a model that would predict customer churn. The Driverless AI model that was generated with the settings shown below and gave us the results below:
+In Task 2, we ran an experiment to build a model that would predict customer churn. The Driverless AI model that was generated with the settings shown above gave us the results below:
```

Comment:
This sentence was referencing the settings that are shown in the table. If you want to refer just to the experiment in the previous task, please consider rewriting the sentence to something such as:
"In Task 2, we ran an experiment to build a model that would predict customer churn. The results for the Driverless AI model that was generated in the previous task are shown below"
By saying "settings shown above," users might start looking for another table or something that summarizes the settings